components/loader/src/circular-loader/circular-loader.js

```diff
@@ -86,8 +69,10 @@ CircularLoader.defaultProps = {
 CircularLoader.propTypes = {
     className: PropTypes.string,
     dataTest: PropTypes.string,
+    invert: PropTypes.bool,
     large: sharedPropTypes.sizePropType,
     small: sharedPropTypes.sizePropType,
+    xsmall: sharedPropTypes.sizePropType,
```

Comment:
I'm not a big fan of abbreviations in code in general..
In this particular case it's quite easy to confuse `xsmall` with `small` as there's only one letter difference and `extrasmall` requires less thinking when reading code

Reply:
Thanks for the tip, that makes a lot of sense. Changed to `extrasmall`.